src/liquid/components/list_cell.html

```diff
@@ -11,6 +11,13 @@
       <a href="https://{{ session.authInfo.domain }}/#/contacts/{{ include.linkTo.id }}" target="_blank">
         {{ include.values[include.property.property_name] }}
       </a>
+    {% elsif include.property.type == 'select_one' %}
+      <span class="tag is-light">{{ include.property.parameter[include.values[include.property.property_name]] }}</span>
```

Comment:
`assign`?  this is a pretty long and crazy line that's hard to read

Reply:
also - is the `tag is-light` needed? no other columns have highlighting or formatting like this?  i'm also not seeing any value displayed when i import something via CSV which doesn't match... shouldn't we see the value from the CSV but with red highlights?

![image](https://github.com/medic/cht-user-management/assets/9014751/1b4dc126-1976-412a-9d2b-73cc7d2e93b4)